.github/workflows/reproducibility.yaml

```diff
@@ -58,6 +77,15 @@ jobs:
         run: |
           ./scripts/docker_run ./scripts/xtask build-oak-functions-server-variants
 
+      # Upload artifacts to Ent, where they will be retained and publicly accessible by their hash.
+      # This only applies to `push` events, since it needs access to a valid Ent API key (which is
+      # not available on forks / PRs).
+      - name: Upload to Ent
+        if: github.event_name == 'push'
+        run: |
+          ent put ./target/x86_64-unknown-linux-musl/release/oak_functions_loader_base
```

Comment:
Will this only run if the step to download ent is successful?

Reply:
Yes, but that step should never fail anyways.